src/data_provider/src/sharedDefs.h

```diff
@@ -21,17 +21,11 @@ constexpr auto WM_SYS_IF_FILE {"/etc/network/interfaces"};
 constexpr auto WM_SYS_IF_DIR_RH {"/etc/sysconfig/network-scripts/"};
 constexpr auto WM_SYS_IF_DIR_SUSE {"/etc/sysconfig/network/"};
 constexpr auto WM_SYS_NET_DIR {"/proc/net/" };
-
 constexpr auto DPKG_PATH {"/var/lib/dpkg/"};
 constexpr auto DPKG_STATUS_PATH {"/var/lib/dpkg/status"};
-
 constexpr auto RPM_PATH {"/var/lib/rpm/"};
-
 constexpr auto PACMAN_PATH {"/var/lib/pacman"};
-
-constexpr auto APK_PATH {"/lib/apk/db"};
```

Comment:
We shouldn't remove this.

Reply:
Same of next comment answered I understand what implementation was changed

Reply:
This is present in master, and shouldn't be modified here. 

![image](https://user-images.githubusercontent.com/13010397/233648757-c402102a-77c7-4a7d-a6ab-2f8cc09c7cf5.png)